README.md

```diff
@@ -102,14 +102,26 @@ sudo apt install -y libgmp3-dev # linux
 brew install gmp # mac
 ```
 
```

Comment:
`nvm install 16` disappeared, but it should still be on the readme

Reply:
noted